src/components/projects/CreateProjectForm.vue

```diff
@@ -917,8 +1063,21 @@ export default {
     this.targets[0].ra = this.mount_ra
     this.targets[0].dec = this.mount_dec
     this.targets[0].name = this.mount_object
-    this.targets.RAhours = true
-    this.targets.hrsminssecs = false
+
+    // initialize smart stack and long stack to camera tab values
+    this.project_constraints.longstack = this.longstackIsActive
+    this.project_constraints.smartstack = this.smartstackIsActive
+  },
+  created () {
+    // initialize expiry date to one lunar month from now, and start date to today
+    const today = new Date()
+
+    this.project_constraints.expiry_date.setDate(today.getDate() + 28)
```

Comment:
Try to avoid [magic numbers](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)